libraries/plugins/rc/resource_count.cpp

```diff
@@ -3,35 +3,238 @@
+   int64_t transaction_object_byte_size       =         STATE_TRANSACTION_BYTE_SIZE;
+
+   // vesting_delegation_object
+   int64_t vesting_delegation_object_base_size = 60*STATE_BYTES_SCALE;
+
+   // vesting_delegation_expiration_object
+   int64_t vesting_delegation_expiration_object_base_size = 44*STATE_BYTES_SCALE;
+
+   // withdraw_vesting_route_object
+   int64_t withdraw_vesting_route_object_base_size = 43*STATE_BYTES_SCALE;
+
+   // witness_object
+   int64_t witness_object_base_size           = 266    *STATE_BYTES_SCALE;
+   int64_t witness_object_url_char_size       =   1    *STATE_BYTES_SCALE;
+
+   // witness_vote_object
+   int64_t witness_vote_object_base_size      = 40     *STATE_BYTES_SCALE;
+};
+
 struct count_operation_visitor
 {
    typedef void result_type;
 
    mutable int32_t market_op_count = 0;
    mutable int32_t new_account_op_count = 0;
+   mutable int64_t state_bytes_count = 0;
 
-   count_operation_visitor() {}
+   const state_object_size_info& _w;
+
+   count_operation_visitor( const state_object_size_info& w ) : _w(w) {}
+
+   int64_t get_authority_byte_count( const authority& auth )const
+   {
+      return _w.authority_base_size
+           + _w.authority_account_member_size * auth.account_auths.size()
+           + _w.authority_key_member_size * auth.key_auths.size();
+   }
 
    template< typename OpType >
    void operator()( const OpType op )const {}
 
-   void operator()( const limit_order_create_operation& )const
-   { market_op_count++; }
-   void operator()( const limit_order_cancel_operation& )const
-   { market_op_count++; }
+   void operator()( const account_create_operation& op )const
+   {
+      state_bytes_count +=
+           _w.account_object_base_size
+         + _w.account_authority_object_base_size
+         + get_authority_byte_count( op.owner )
+         + get_authority_byte_count( op.active )
+         + get_authority_byte_count( op.posting );
+      new_account_op_count++;
+   }
+
+   void operator()( const account_create_with_delegation_operation& op )const
+   {
+      state_bytes_count +=
+           _w.account_object_base_size
+         + _w.account_authority_object_base_size
+         + get_authority_byte_count( op.owner )
+         + get_authority_byte_count( op.active )
+         + get_authority_byte_count( op.posting )
+         + _w.vesting_delegation_object_base_size;
+      new_account_op_count++;
+   }
+
+   void operator()( const account_witness_vote_operation& op )const
+   { state_bytes_count += _w.witness_vote_object_base_size;       }
+
+   void operator()( const comment_operation& op )const
+   {
+      state_bytes_count +=
+           _w.comment_object_base_size
+         + _w.comment_object_permlink_char_size * op.permlink.size()
+         + _w.comment_object_parent_permlink_char_size * op.parent_permlink.size()
+         ;
+   }
+
+   void operator()( const comment_payout_beneficiaries& bens )const
```

Comment:
There is no refund of cost for deleting an account. Was this overlooked or a limitation of the rc plugin?